brmp/fit.py

```diff
@@ -177,6 +213,26 @@ def compute_diag_or_default(diag, samples):
 # be to do this without performing any unnecessary slicing. (Though
 # this sounds fiddly.)
 def marginals(fit, qs=default_quantiles):
+    """Produces a table containing statistics of the marginal
+    distibutions of the parameters of the fitted model.
+
+    :param fit: A model fit.
+    :type fit: brmp.fit.Fit
+    :param qs: A list of quantiles to include in the output.
+    :type qs: list
+    :return: A table of marginal statistics.
+    :rtype: brmp.fit.ArrReprWrapper
+
+    Example::
```

Comment:
I just had a more general question - is there a reason why functions like `marginals` and `get_scalar_params` are not methods of the `Fit` class?

Reply:
No, there's nothing preventing us making this change. (And there's probably a bunch more non-idiomatic python waiting to be uncovered!)